Vungle/src/main/java/com/mopub/mobileads/VungleRewardedVideo.java

```diff
@@ -237,14 +244,24 @@ private void modifyAdConfig(AdConfig adConfig, VungleMediationConfiguration medi
      */
     private class VungleRewardedRouterListener implements VungleRouterListener {
         @Override
-        public void onAdEnd(String placementId) {
-            if (mPlacementId.equals(placementId)) {
-                MoPubLog.log(getAdNetworkId(), CUSTOM, ADAPTER_NAME, "onAdEnd - Placement ID: " + placementId);
+        public void onAdEnd(@NonNull String placementReferenceId) {
+            if (mPlacementId.equals(placementReferenceId)) {
+                MoPubLog.log(getAdNetworkId(), CUSTOM, ADAPTER_NAME, "onAdEnd - Placement ID: " +
+                        placementReferenceId);
                 mIsPlaying = false;
 
-                if (mInteractionListener != null) {
-                    mInteractionListener.onAdDismissed();
-                }
+                MoPubLog.log(getAdNetworkId(), SHOULD_REWARD, ADAPTER_NAME, MoPubReward.NO_REWARD_AMOUNT,
+                        MoPubReward.NO_REWARD_LABEL);
+                // Vungle does not provide a callback when a user should be rewarded.
+                // You will need to provide your own reward logic if you receive a reward with
+                // "NO_REWARD_LABEL" && "NO_REWARD_AMOUNT"
+                MoPubRewardedVideoManager.onRewardedVideoCompleted(VungleRewardedVideo.class,
```

Comment:
as there is separate onRewardedVideoClosed. , should the onRewardedVideoCompleted be called from onAdRewarded.

Reply:
We can remove code parts from line 253 to line 261.